addon/templates/components/markers-styles/image/icon-editor.hbs

```diff
@@ -2,15 +2,28 @@
   <thead>
     <tr>
       <th class="six wide column">
-        {{iconCaption}}
+        
+        {{iconCaption}}        
+        
+        {{#if allowDisabling}}
```

Comment:
Code must be formatted:
`{{#if allowDisabling}}`
`..{{flexberry-checkbox`
`....value=_disabled`
`....class="rightFloated"`
`..}}`
`{{/if}}`